web/src/todo_widget.ts

```diff
@@ -39,59 +114,59 @@ export class TaskData {
         }
 
         for (const [i, data] of tasks.entries()) {
-            this.handle.new_task.inbound("canned", {
+            this.handle.new_task.inbound(message_sender_id, {
                 key: i,
```

Comment:
Normally, the `inbound` function receives the `sender_id` argument as number. But here, they have mentioned `canned` here. I have changed that to `message_sender_id`.

Is this correct? Or should I be changing something else?

Reply:
Change the type annotations of the parameter of the `inbound` function to accept `string | number`, otherwise, it will result in a key error. You can read the [CZO thread](https://chat.zulip.org/#narrow/stream/9-issues/topic/.E2.9C.94.20poll.20responses.20removed.20or.20missing.3F) for more context.

Reply:
Also, it is recommended to create a separate commit when making these types of changes.